protos/pointerdb/pointerdb.proto

```diff
@@ -65,12 +65,13 @@ message Pointer {
 
   bytes inline_segment = 3;
   RemoteSegment remote = 4;
-  int64 size = 5;
+  int64 segment_size = 5;
+  int32 stripe_size = 6;
```

Comment:
This new field should be part of the RedundancyScheme, not the Pointer.

So there won't be any need to rename the `size` field to `segment_size`.

And the new field should not be `stripe_size`, but `erasure_share_size`, regardless of what's written in the JIRA ticket. It is the erasure share size that is a parameter of the RS scheme, not the stripe size, although they are related.